packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -296,6 +302,106 @@ Returning an error object will cause Okta to return an OAuth 2.0 error to the re
+
+See [ID Token Claims](https://developer.okta.com/docs/api/resources/oidc/#id-token-claims) for a list of ID token reserved claims that you can't modify.
+
+```json
+{
+   "commands": [ 
+        { 
+            "type": "com.okta.identity.patch",
+            "value": [ 
+                 { 
+                     "op": "replace",
+                     "path": "/claims/extPatientId",
+                     "value": "1234"
+                  }
+             ] 
+         }, 
+         { 
+             "type": "com.okta.access.patch",
+             "value": [
+                  { 
+                     "op": "replace",
+                     "path": "/claims/external_guid",
+                     "value": "F0384685-F87D-474B-848D-2058AC5655A7" 
+                   }
+              ] 
+          }
+      ]
+}
+```
+
+### Sample Response to Modify Token Lifetime
+You can modify how long the Access and ID tokens are valid by specifying the `lifetime` in seconds. The `lifetime` value must be a minimum of five minutes (300 seconds) and a maximum of 24 hours (86,400 seconds).
+
+```json
+{
+   "commands": 
+     [ 
+        { 
+            "type": "com.okta.identity.patch",
+            "value": [ 
+                 { 
+                     "op": "replace",
+                     "path": "/token/lifetime/expiration",
+                     "value": 36000
+                  }
+             ] 
+         }, 
+         { 
+             "type": "com.okta.access.patch",
+             "value": [
+                  { 
+                     "op": "replace",
+                     "path": "/token/lifetime/expiration",
+                     "value": 36000 
+                   }
+              ] 
+          }
+      ]
+}
+```
+### Sample Response to Remove Token Claims
+You can remove existing custom claims or OIDC standard profile claims, such as `birthdate` or `locale`. You can't, however, remove any system-specific claims, such as `iss` or `ver`, and you can't remove a claim that isn't currently part of the token in the request payload. Attempting to remove a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events. 
+
+For the list of Access token reserved claims that you can't remove, see [Access Tokens Scopes and Claims](https://developer.okta.com/docs/api/resources/oidc/#access-token-scopes-and-claims). Note that although the `aud` claim is listed as a reserved claim, you can remove that claim in a response.
+
+See [ID Token Claims](https://developer.okta.com/docs/api/resources/oidc/#id-token-claims) for a list of ID token reserved claims that you can't remove.
+
+> Note: For the `remove` operation, the `value` property isn't required and should be set to `null`. Providing any other value fails the entire PATCH response.
```

Comment:
I was thinking that we should somehow specify that `value` must be set to `null` only if customer sends it in response. May be something like this:
`For the `remove` operation, the `value` property isn't required but should be set to `null` if provided within response`